cmd/fossa/cmd/report/report.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/fossas/fossa-cli/module"
 )
 
-var (
+const (
 	Unknown = "show-unknown"
```

Comment:
[nit] Can we move this into `licenses.go` and put it right next to `JSON`? Both are constants only used by `licenses.go`.